lib/kubeclient.rb

```diff
@@ -17,7 +17,7 @@ module Kubeclient
   # Kubernetes Client
   class Client
     attr_reader :api_endpoint
-    ENTITIES = %w(Pod Service ReplicationController Node Event Endpoint)
+    ENTITY_TYPES = %w(Pod Service ReplicationController Node Event Endpoint)
```

Comment:
> We recently added Namespaces, is it missing now?

this PR is up to date to yesterday, namespaces were added today. it supposed to rebase with np on top of namespaces PR imho.